lecture10 template file/lecture10-vpc.outputs.yaml

```
@@ -0,0 +1,169 @@
+      Tags:
+        -
+          Key: Name
+          Value: PublicSubnet1
+  
+  WebPublicSubnet2:
+    Type: AWS::EC2::Subnet
+    Properties:
+      AvailabilityZone: ap-northeast-1c
+      VpcId: !Ref VPC
+      CidrBlock: 10.0.16.0/20
+      MapPublicIpOnLaunch: true
+      Tags:
+        -
+          Key: Name
+          Value: PublicSubnet2
+
+  privatesubnet1:
+    Type: AWS::EC2::Subnet
+    Properties: 
+      AvailabilityZone: ap-northeast-1a
+      CidrBlock: 10.0.128.0/20
+      MapPublicIpOnLaunch: true
+      Tags:
+        -  Key: Name
+           Value: privatesubnet1
+      VpcId: !Ref VPC
+
+  privatesubnet2:
+   Type: AWS::EC2::Subnet
+   Properties: 
+      AvailabilityZone: ap-northeast-1c
+      CidrBlock: 10.0.144.0/20
+      MapPublicIpOnLaunch: true
+      Tags:
+        - Key: Name
+          Value: privatesubnet2
+      VpcId: !Ref VPC
+    
+  rdssubnetgroup: 
+   Type: AWS::RDS::DBSubnetGroup
+   Properties: 
+      DBSubnetGroupDescription: rdssubnetgroup
+      SubnetIds:  
+        - !Ref privatesubnet1
+        - !Ref privatesubnet2
+      DBSubnetGroupName: lecture10
+      Tags: 
+      - Key: Name
+        Value: rdssubnetgroup
+  
+
+# Internet Gateway
+#
+#
+  InternetGateway:
+    Type: AWS::EC2::InternetGateway
+
+  InternetGatewayAttachment:
+    Type: AWS::EC2::VPCGatewayAttachment
+    Properties:
+      InternetGatewayId: !Ref InternetGateway
+      VpcId: !Ref VPC
+
+# Routetable
+#
+#
+  RouteTable:
+    Type: AWS::EC2::RouteTable
+    Properties:
+      VpcId: !Ref VPC
+
+  DefaultRoute:
+    Type: AWS::EC2::Route
+    Properties:
+      RouteTableId:
+        Ref: RouteTable
+      DestinationCidrBlock: 0.0.0.0/0 
+      GatewayId: 
+        Ref: InternetGateway
+    DependsOn: 
+      - SubnetRouteTableAssociation
+
+  SubnetRouteTableAssociation:
+    Type: AWS::EC2::SubnetRouteTableAssociation
+    Properties:
+      RouteTableId: !Ref RouteTable
+      SubnetId: !Ref WebPublicSubnet1
+      
+  privateroutetable:
+   Type: AWS::EC2::RouteTable
+   Properties:
+    Tags:
+      - Key: Name
+        Value: privateroutetable
+    VpcId: !Ref VPC
+
+  routetableattach1:
+   Type: AWS::EC2::SubnetRouteTableAssociation
+   Properties: 
+    RouteTableId: !Ref privateroutetable
+    SubnetId: !Ref privatesubnet1
+
+  routetableattach2:
+   Type: AWS::EC2::SubnetRouteTableAssociation
+   Properties: 
+    RouteTableId: !Ref privateroutetable
+    SubnetId: !Ref privatesubnet2
+
+
+# エクスポートするためのセクション
+Outputs:
+  VPC:
+    Value: !Ref VPC  # 作成されたIDを
+    Export:
+      Name: VPCID  # この名前でエクスポートする
+  
+  WebPublicSubnet1:
+    Value: !Ref WebPublicSubnet1  # 作成されたIDを
+    Export:
+      Name: WebPublicSubnet1ID  # この名前でエクスポートする
+
+  WebPublicSubnet2:
+    Value: !Ref WebPublicSubnet2  # 作成されたIDを
+    Export:
+      Name: WebPublicSubnet2ID  # この名前でエクスポートする
+
+  rdssubnetgroup:
+    Value: !Ref rdssubnetgroup  # 作成されたIDを
+    Export:
+      Name: rdssubnetgroup  # この名前でエクスポートする
```

これだけ命名規則に整合性がない？
これも何が格納される？
```suggestion
      Name: RdsSubnetGroup  # この名前でエクスポートする
```

---

以下に修正
  rdssubnetgroup:
    Value: !Ref rdssubnetgroup  # 作成されたIDを
    Export:
      Name: RdsSubnetGroupMysqlID  # この名前でエクスポートする